### PR TITLE
Add CSP nonce to CSS as well

### PR DESCRIPTION
**Pull request by rullzer**
Further tightening of our CSP.

It seems we need to export the nonce now to all the webpack code that wants to inject CSS (not just chunks).

### CODE REVIEW

File: core/src/login.js [hunk 1]
```diff
@@ -22,6 +22,8 @@
 import Vue from 'vue';
 import queryString from 'query-string';
 
+__webpack_nonce__ = btoa(OC.requestToken)
```
Comment (rullzer) [inline core/src/login.js]: @juliushaertl @skjnldsv I assumed this was enough as we use it in other places as well... but it seems not...

Comment (skjnldsv) [reply core/src/login.js]: https://github.com/webpack-contrib/style-loader/issues/306 ?

Comment (skjnldsv) [reply core/src/login.js]: https://github.com/webpack-contrib/style-loader/pull/319

Comment (rullzer) [reply core/src/login.js]: Ok got it. it is because of the import reshuffling.... so the nonce gets set to late

Comment (rullzer) [reply core/src/login.js]: Stupid wrapping seems to be the easy fix...

File: core/src/login.js [hunk 2]
```diff
@@ -1,65 +1,3 @@
...
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-import Vue from 'vue';
-import queryString from 'query-string';
-
 __webpack_nonce__ = btoa(OC.requestToken)
```
Comment (skjnldsv) [inline core/src/login.js]: `import OC from './OC/index'` ?

**Comment by rullzer** [comment]
Ok nevermind. We are by far not ready for this. As it doesn't allow setting 

`style="......"` on divs anymore

**Pull request closed without merging**